.travis.yml

```diff
@@ -15,7 +15,7 @@ os:
 go_import_path: github.com/kata-containers/runtime
 
 go:
-  - "1.10.x"
+  - "1.11.x"
```

Comment:
@gnawux @teawater is this change something you guys agree to or would you prefer i move this out to a seperate patch where we update the baseline for all kata  code-bases?